provider/lxd/credentials.go

```diff
@@ -202,16 +202,20 @@ func (p environProviderCredentials) detectRemoteCredentials(certPEM, keyPEM []by
 	for name, remote := range config.Remotes {
 		if remote.Protocol == lxdnames.ProviderType {
 			certPath := filepath.Join(configDir, "servercerts", fmt.Sprintf("%s.crt", name))
+			authConfig := map[string]string{
+				credAttrClientCert: string(certPEM),
+				credAttrClientKey:  string(keyPEM),
+			}
```

Comment:
Total nit, but the `develop` version has a blank line here -- might make this easier to forward-merge?

Reply:
this is new code which is not in develop yet. regardless, i can deal with any minor issue